CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+## v0.23.0 (2023-12-18)
+
+## What's Changed
+
+Release highlights:
```

Comment:
perhaps summarize this and focus on what may be relevant for operators/users? 

wondering about the implications of:
> Add new DB column `messageHash`

does this break compatibility in any way? just in case we need to ⚠️

Reply:
Good point! I've added the :warning: symbol and reduced the highlights as you and @gabrielmer mentioned